src/App/App.js

```diff
@@ -14,19 +16,52 @@ function App() {
     setSelectedPokemon(null);
   };
 
+  const handleAddToTeam = () => {
+    if (selectedPokemon) {
+      const updatedTeam = [...team];
+      const emptySlotIndex = updatedTeam.findIndex(pokemon => pokemon === null);
+      if (emptySlotIndex !== -1) {
+        updatedTeam[emptySlotIndex] = selectedPokemon;
+        setTeam(updatedTeam);
+        console.log(team)
+      } else {
+        console.log("Your team is full. Remove a Pokemon to add a new one.");
+      }
+    }
+  };
+
+  const handleRemoveFromTeam = (index) => {
+    const updatedTeam = [...team];
+    updatedTeam[index] = null;
+    setTeam(updatedTeam);
+  };
+
   return (
     <div className="App">
       <header className="App-header">
-        {/* Your existing header content */}
+        <h1>PokéLex</h1>
```

Comment:
Dont forget to implement routing - I really wanna see the polish on this showcase app!